gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -91,6 +179,9 @@ struct Settings {
     index_max_nanos: u64,
     allow_create_scope: bool,
     keycloak_file: Option<String>,
+    retention_type: Option<String>,
+    retention_days: f64,
```

Comment:
This should be `Option<f64>` so that we can tell when the parameter is not specified or specified as 0. Same with `retention_bytes`.

Reply:
done